src/api/app/views/webui/users/tokens/show.js.erb

```diff
@@ -0,0 +1,3 @@
+$('#token-secret').html("<%= escape_javascript(@token.string) %>");
```

Comment:
I removed the `CopyToClipboardInputComponent` and went by a plain text because the component does not support several instances in the same page.

Reply:
This is also fixed